tft_bot/economy/ocr_standard.py

```diff
@@ -0,0 +1,49 @@
+"""
+Module holding the OCR standard economy mode.
+"""
+import cv2
+import mss
+import numpy
+from pytesseract import pytesseract
+
+from .base import EconomyMode
+
+
+class OCRStandardEconomyMode(EconomyMode):
+    """
+    OCR standard economy mode implementation.
+    """
+
+    def __init__(self, wanted_traits: list[str], prioritized_order: bool, tesseract_location: str):
+        super().__init__(wanted_traits, prioritized_order)
+        pytesseract.tesseract_cmd = tesseract_location
+
+    def loop_decision(self, minimum_round: int):
+        self.purchase_units(amount=3)
```

Comment:
Would this not put the purchase below 50 gold, potentially? 🤔 

Would it maybe make sense to do some sort of round-to-gold minimum?

Reply:
Yes it would, but I believe that going below 50 for a unit purchase is very acceptable. For example, we roll at 55 which puts us at 53 and see a 4-cost unit in the shop, we obviously want to buy that. Same for any unit showing up until we have 50 gold. 

Obviously this is very opinionated, but that's why I tried to make the economy mode base class so everyone can add their own economy mode they think is better :) So for this standard mode I'd say the logic is fine.

Reply:
I don't feel strongly enough about this to block it, I think that makes enough sense so works for me 👌